app/scripts/lib/metrics.js

```diff
@@ -468,10 +481,18 @@ define(function (require, exports, module) {
       if (flowId !== this._flowId) {
         this._flowId = flowId;
         this._flowBeginTime = flowBeginTime;
-        this.logEvent(`flow.${viewName}.begin`);
+        this.logFlowEvent('begin', viewName);
       }
     },
 
+    logFlowEvent (event, viewName) {
```

Comment:
in [view->base.js](https://github.com/mozilla/fxa-content-server/pull/4317/files#diff-af500bac5338814fda67ec4de209ac46R650), you use `eventName` and `viewName`, maybe make the names consistent here?